lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -1753,6 +1753,23 @@ defmodule Kernel.SpecialForms do
   catch values also support `:error`, as in Erlang, although it is
   commonly avoided in favor of raise/rescue control mechanisms.
 
+  ## After clauses
+
+  An `after` clause allows you to define cleanup logic that will be invoked both
+  when the tried block of code succeeds and also when an error is raised. Note,
+  however, that if a linked process exits in the `try` block, the process will
+  immediately exit and your `after` clause will not get executed. Luckily, most
+  resources in Elixir--such as open files, ETS tables, ports, sockets, etc--are
+  implemented as linked processes that will automatically clean themselves up if
```

Comment:
ETS tables, ports, usually sockets and sometimes files are not processes. It would be better to say "are linked to or monitor the owning process and will automatically clean themselves if that process exits."